integration-tests/test/conftest.py

```diff
@@ -120,14 +119,20 @@ def random_generator(command_line_options: CommandLineOptions) -> Generator[Rand
 
 
 @contextlib.contextmanager
-def testing_context(command_line_options: CommandLineOptions, random_generator: Random, docker_client: DockerClient, bootstrap_key: PrivateKey = None, peers_keys: List[PrivateKey] = None, network_peers: int = 2) -> Generator[TestingContext, None, None]:
+def testing_context(command_line_options: CommandLineOptions, random_generator: Random, docker_client: DockerClient, bootstrap_key: PrivateKey = None, peers_keys: List[PrivateKey] = None, network_peers: int = 2, validator_bonds_dict: Dict[PrivateKey, int] = None) -> Generator[TestingContext, None, None]:
```

Comment:
The `validator_bonds_dict` isn't used anywhere, right?

Reply:
yes~

Reply:
So I'd say this PR is too small to be merged.  It only makes sense to merge it once the added code is in use.  Please push more changes to it related to whatever you're aiming at.